### DISCUSSION TITLE
Unable to display

### DISCUSSION THREAD

**Discussion by Xsatc77** [gotson/komga#1210] [General]
I used docker-compose and got komga up and running.  The problem is when I go to the address where it should be, I get a Problem Loading Page error, Unable to connect, Firefox can’t establish a connection to the server at 192.168.1.78:8080.  The logs look okay, no errors displayed.  Maybe Im missing something.  Attaching the logfile.
[_komga_logs(2).txt](https://github.com/gotson/komga/files/12541521/_komga_logs.2.txt)

**Comment by gotson** [comment]
Please open an issue instead of a discussion.
